modules/reference/pages/console/record-deserialization.adoc

```diff
@@ -1,27 +1,35 @@
-= Record Deserialization
-:description: Learn how record deserialization in Redpanda Console works and what encodings it supports.
+= Deserialization in Redpanda Console
+:description: Learn how Redpanda Console deserializes messages.
 :page-aliases: console:features/record-deserialization.adoc
 
-When consuming messages, Redpanda Console's message viewer is capable of automatically deserializing messages.
-It identifies the correct deserialization type by trying to decode the record's key, value, or header with all
-available deserialization methods. The supported deserialization methods are:
+In Redpanda, the messages exchanged between producers and consumers contain raw bytes. Schemas work as an agreed-upon format, like a contract, for producers and consumers to serialize and deserialize those messages. If a producer breaks this contract, consumers can fail. 
 
-* Kafka's internal binary formats (for example the `__consumer_offsets` topic )
+Redpanda Console automatically tries to deserialize incoming messages and show them in human-readable format. It tries different deserializers and lets you choose the deserializer before producing the message in the desired format. If the deserialization fails, it provides troubleshooting information. For example, deserialization requires fetching schemas from a schema registry, and that fetch could fail. If downstream applications crash, you can find which message caused the crash, check the message content, and see the schema that was used in deserializing it. You can also download the raw bytes of the message to feed it directly to your client deserializer or share it with support.
```

Comment:
> If the deserialization fails, it provides troubleshooting information.

Maybe worth to clarify what failing here means. As described we try a bunch of deserialization strategies in a specific order until we found a strategy that doesn't yield an error. The very last strategy is called "binary" and in that case we would show the bytes as they are stored in a hex viewer. Sometimes it may be expected to see the payload just in hex bytes (e.g. because the payload is encrypted or because it uses a serializer that we don't support to deserialize). In that case we always show the troubleshoot report though.

Reply:
> If downstream applications crash, you can find which message caused the crash, check the message content, and see the schema that was used in deserializing it.

I think this may be a bit misleading. Maybe reword to something that expresses that this may be used as a tool to debug applications that consume messages from that topic. I remember that I provided the deserialization issue an example, but this was a very specific example I had experienced at my past co.

Reply:
Thanks @weeco. Please review this updated section: https://deploy-preview-175--redpanda-docs-preview.netlify.app/23.3/reference/console/record-deserialization/